pkg/settings/setting_test.go

```diff
@@ -54,3 +54,25 @@ func TestSystemDefaultRegistryDefault(t *testing.T) {
 	}
 
 }
+
+// TestSystemFeatureChartRefreshSecondsDefault tests that the default registry is either
```

Comment:
I think this should be system feature chart refresh seconds instead of default registry

Reply:
You're right! Just changed it